config.go

```diff
@@ -404,6 +412,9 @@ func loadConfig() (*config, error) {
 			RejectCacheSize:  channeldb.DefaultRejectCacheSize,
 			ChannelCacheSize: channeldb.DefaultChannelCacheSize,
 		},
+		Watchtower: &lncfg.Watchtower{
+			TowerDir: defaultTowerDir,
```

Comment:
If the user provides a different `lnddir`, the `datadir` will be in a different location, so we'll need to handle this case correctly. This is done a bit further down after parsing all the flags. We'll also need to use `cleanAndExpandPath` on it.

Reply:
nice catch, i think it's been addressed in the current branch. lmk if there's anything else you see